src/main/java/com/julienvey/trello/impl/TrelloImpl.java

```diff
@@ -49,31 +50,36 @@
 import static com.julienvey.trello.impl.TrelloUrl.GET_MEMBER_CARDS;
 import static com.julienvey.trello.impl.TrelloUrl.GET_ORGANIZATION_BOARD;
 import static com.julienvey.trello.impl.TrelloUrl.GET_ORGANIZATION_MEMBER;
-import static com.julienvey.trello.impl.TrelloUrl.REMOVE_MEMBER_FROM_CARD;
 import static com.julienvey.trello.impl.TrelloUrl.REMOVE_MEMBER_FROM_BOARD;
+import static com.julienvey.trello.impl.TrelloUrl.REMOVE_MEMBER_FROM_CARD;
 import static com.julienvey.trello.impl.TrelloUrl.UPDATE_CARD;
-import static com.julienvey.trello.impl.TrelloUrl.UPDATE_LABEL;
 import static com.julienvey.trello.impl.TrelloUrl.UPDATE_CARD_COMMENT;
+import static com.julienvey.trello.impl.TrelloUrl.UPDATE_LABEL;
 import static com.julienvey.trello.impl.TrelloUrl.createUrl;
 import static com.julienvey.trello.impl.TrelloUrl.createUrlWithNoArgs;
 
-import com.julienvey.trello.domain.AddMemberToBoardResult;
-import com.julienvey.trello.domain.Label;
-
-import java.util.*;
-
-import com.julienvey.trello.ListNotFoundException;
-import com.julienvey.trello.NotFoundException;
-import com.julienvey.trello.TrelloBadRequestException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
```

Comment:
I see a lot of changed imports. let's preserve whatever imports style is there for now and just perform a one-time reformat separately to avoid extra changes in PRs.
formatting templates can be different for different people, so I am worried this may get changed too many times.
I don't want to deal with a custom formatter at this point.